saleor/static/dashboard/scss/components/_google-preview.scss

```diff
@@ -0,0 +1,42 @@
+.google-preview {
+    text-align: left;
+    letter-spacing: initial;
+
+    .container {
```

Comment:
To be consistent with existing codebase, we should use BEM naming convention - in this case `.google-preview` is our component name, and `.container` should be renamed to `.google-preview__container` ( in `.scss` to `&__container` ) to indicate that it belongs to `google-preview` component.